denguefever_tw/dengue_linebot/utils.py

```diff
@@ -5,6 +5,7 @@
 from time import sleep
 from selenium import webdriver
 from pyvirtualdisplay import Display
+from urllib.parse import urljoin
```

Comment:
You should import as the following sequence
```python
from django.conf import settings

import logging
from urllib.parse import urljoin
from time import sleep

import requests
from selenium import webdriver
from pyvirtualdisplay import Display
from linebot.exceptions import LineBotApiError
from linebot.models import TextSendMessage, ImageSendMessage

from .decorators import log_line_api_error
```

Reply:
But in `denguebotfsm.py` , why?
```
import logging
import requests
from datetime import datetime
from functools import partial
from urllib.parse import parse_qs, urljoin

from geopy.geocoders import GoogleV3
from condconf import CondMeta, cond_func_generator

```

Reply:
Oh, I miss it.

It should be 
```python
import logging
from datetime import datetime
from functools import partial
from urllib.parse import parse_qs, urljoin

import requests
from geopy.geocoders import GoogleV3
from condconf import CondMeta, cond_func_generator
```

Although it's a minor issue, it is still great to handle every detail.
Especially for an open source project.

Reply:
I see! Fixed it.